<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_ordemservico</class>
 <widget class="QDialog" name="frm_ordemservico">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ordem de Serviço</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>811</width>
     <height>631</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Nova O.S.</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>139</width>
       <height>50</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>M. Pagamaneto</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_meiopagamento"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_5">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>320</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Data Entrada</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit"/>
      </item>
     </layout>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>251</width>
       <height>211</height>
      </rect>
     </property>
     <property name="title">
      <string>Dados Veículo:</string>
     </property>
     <widget class="QLineEdit" name="txt_nomeVeiculo">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>76</y>
        <width>201</width>
        <height>25</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Modelo Veículo</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_placaVeiculo">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>112</y>
        <width>123</width>
        <height>25</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Placa Veículo</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_corVeiculo">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>149</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Cor Veículo</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txt_proprietarioVeiculo">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>201</width>
        <height>25</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Cliente Proprietário</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_selecionaveiculo">
      <property name="geometry">
       <rect>
        <x>153</x>
        <y>180</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Selecionar</string>
      </property>
     </widget>
    </widget>
    <widget class="QPlainTextEdit" name="Ptxt_garantiaservico">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>420</y>
       <width>221</width>
       <height>111</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>397</y>
       <width>121</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Garantia Serviço</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>663</x>
       <y>568</y>
       <width>89</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Gerar O.S.</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>8</y>
       <width>481</width>
       <height>241</height>
      </rect>
     </property>
     <property name="title">
      <string>Peças:</string>
     </property>
     <widget class="QTableWidget" name="tw_listapecas">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>461</width>
        <height>201</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>289</y>
       <width>141</width>
       <height>50</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>KM Veículo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txt_kmVeiculo"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>146</width>
       <height>108</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>V. Total Serviço</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_2"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>V. Total Mercadoria</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_3"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <zorder>groupBox_2</zorder>
    <zorder>layoutWidget</zorder>
    <zorder>layoutWidget_2</zorder>
    <zorder>layoutWidget_3</zorder>
    <zorder>layoutWidget_5</zorder>
    <zorder>layoutWidget_6</zorder>
    <zorder>groupBox</zorder>
    <zorder>label_2</zorder>
    <zorder>txt_kmVeiculo</zorder>
    <zorder>Ptxt_garantiaservico</zorder>
    <zorder>label_5</zorder>
    <zorder>layoutWidget</zorder>
    <zorder>pushButton</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Gestão O.S.</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
